ui/js/pods-dfv/_src/pick/views/select-view.js

```diff
@@ -230,12 +230,18 @@ export const SelectView = PodsMn.CollectionView.extend( {
 	},
 
 	/**
-	 * @var {RelationshipCollection} this.collection
 	 *
 	 * @returns {boolean} true if unlimited selections are allowed or we're below the selection limit
 	 */
 	validateSelectionLimit: function () {
 		let limit, numSelected;
+		const format_type = this.fieldConfig.pick_format_type;
+		const format_single = this.fieldConfig.pick_format_single;
+
+		// Selection limit should be clear if the placeholder is selected in a single-select dropdown
+		if ( "" === this.$el.val() && "single" === format_type && "dropdown" === format_single ) {
```

Comment:
@pglewis Should we remove the `format_single` conditional check here?

Reply:
Is there any potential harm or side effect of it being there?  I was just trying to be explicit as possible (read: careful to avoid jostling anything else known to be working)

Reply:
Phrased another way: if there is no harm to it being there other than possibly being redundant, I'd rather err on the side of safety.

Reply:
Would this still cause a problem for radio and single list view?

Reply:
The problem in this case is specifically when selecting the "-- Select One --" placeholder.